src/Engine/Objects/Player.cpp

```diff
@@ -7607,20 +7455,22 @@ MERCHANT_PHRASE Player::SelectPhrasesTransaction(ItemGen *pItem, BuildingType bu
     multiplier = p2DEvents[BuildID_2Events - 1].fPriceMultiplier;
     switch (ShopMenuType) {
         case 2:
-            price = GetBuyingPrice(itemValue, multiplier);
+            price = PriceCalculator::getItemBuyingPriceForPlayer(this, itemValue, multiplier);
             break;
         case 3:
             // if (pItem->IsBroken())
             // price = 1;
             // else
-            price = this->GetPriceSell(*pItem,
-                                       multiplier);  // itemValue, multiplier);
+            price = PriceCalculator::getItemSellingPriceForPlayer(
+                this, *pItem,
+                multiplier);  // itemValue, multiplier);
```

Comment:
We don't impose a strict limit on line length, 120 chars is perfectly OK, and around 160 is the comfortable limit. So given a choice – please write these calls in a single line.

Also, you can drop the comment here, it's a remnant from years ago.

Reply:
Oh, I'm so glad line length is not so strict, due to this and 2 spaces indentation I was seriously doubting google style guide :)

Reply:
BTW, are there other unwritten rules (in HACKING.md) which are different than Google's?

Reply:
Just look at the code around and try to match it style-wise, that's pretty much it =).